### ISSUE TITLE
new incompatibility on profile page?

### ISSUE THREAD

**Issue by movAX13h** [patuwwy/ShaderToy-Chrome-Plugin#52]
I'm getting (chrome, win8):
```
Uncaught TypeError: Cannot read property 'appendChild' of null
    at ShaderTile.createHTML (shadertoy-plugin-profile.js:226)
    at new ShaderTile (shadertoy-plugin-profile.js:204)
    at TilesView.parseRow (shadertoy-plugin-profile.js:183)
    at Array.map (<anonymous>)
    at TilesView.getShaders (shadertoy-plugin-profile.js:179)
    at TilesView.waitForShaders (shadertoy-plugin-profile.js:162)
```
when I visit my profile page and it does not fill the sections (headlines visible above list).
Might come from a shadertoy update, not sure.

**Comment by patuwwy** [comment]
It should work again in version 0.9.11.
What version of plugin do You have?

**Comment by movAX13h** [comment]
still the same with v0.9.11

**Comment by patuwwy** [comment]
Fixed in 09.12 - shaders' statuses are not hardcoded anymore to english
